src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -654,11 +655,8 @@ export default {
 		canStar() {
 			return this.isStarred !== null
 		},
-		hasFigure() {
-			return this.$slots.header || this.background
-		},
 		hasFigureClickListener() {
-			return this.$listeners['figure-click']
+			return this.$attrs['figure-click']
```

Comment:
This does not work anymore. Also see https://github.com/nextcloud-libraries/nextcloud-vue/pull/3692/files#r1276581301.

Reply:
Fixed in 7999dada18934a0416718fe8576fc362c05b8721 by removing `figure-click` from `emits` and use it as `$attrs.onFigureClick(event)` as suggested by @ShGKme.